### ISSUE TITLE
bug(mat-chip-grid): impossible to navigate to the input using VoiceOver commands when input is not empty

### ISSUE THREAD

**Issue by 6apc1k** [angular/components#30316]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

When nesting input inside of the mat-chip-grid, it becomes inaccessible with the screenreader commands. 

### Reproduction

StackBlitz link: https://stackblitz.com/run?file=src%2Fexample%2Fchips-input-example.html
Steps to reproduce:
1. Use material chips with input, where input is nested inside of mat-chip-grid.
2. Add at least one chip to the input.
3. Start VoiceOver and focus the form field.
4. Navigate inside the form field using VoiceOver commands (e.g. VO-Right Arrow to move to the next item)

### Expected Behavior

It should be possible to focus the input element when pressing VO-Right Arrow inside the mat-chip-grid. Similar to how it is in chip-autocomplete example, when input is not inside the mat-chip-grid:
https://stackblitz.com/run?file=src%2Fexample%2Fchips-autocomplete-example.css

### Actual Behavior

Only chips and delete buttons can be focused using VoiceOver commands. 

### Environment

- Angular: 19.0.5
- CDK/Material: 19.0.5
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): MacOS

**Comment by SetraArt** [comment]
Hello
The same problem exists with NVDA. 
Do you think it is due to the html attributes used on the component? 

Best regards
